dhash/database.py

```diff
@@ -102,6 +103,36 @@ def dump(self) -> dict:
         }
 
 
+class HashConfig(database.base):
+    """Stores config in format key:value"""
+
+    __tablename__ = "fun_dhash_config"
+
+    key = Column(String, primary_key=True)
+    value = Column(String)
+
+    @staticmethod
+    def get(key: str, value=None) -> str:
+        config = session.query(HashConfig).filter_by(key=key).one_or_none()
+
+        if config:
+            return config.value
+        else:
+            return value
+
+    @staticmethod
+    def set(key: str, value: str):
+        config = session.query(HashConfig).filter_by(key=key).one_or_none()
+
+        if not config:
+            config = HashConfig(key=key)
+
+        config.value = value
+
+        session.merge(config)
+        session.commit()
+
```

Comment:
Each of our database objects must have `__repr__()` and `dump()` functions.

Reply:
Forget about that, gonna fix it.